### PR TITLE
Refactoring Templates

### PR DESCRIPTION
**Pull request by theethaj**
I'm already refactor all templates according to our reviewing guideline. Please review them all especially question-set-detail.html and history.html to make sure that they are up-to-date file.

### CODE REVIEW

File: quizer_game/templates/quizer_game/edit-question-set.html [hunk 1]
```diff
@@ -5,28 +5,29 @@
...
+	  </div>
   </header>
   <h1>Edit Quiz</h1>
 
   <div class="container">
     {% if messages %}
-	    <ul class="messages" id="message">
+	    <ul class="messages" id="message_success">
         {% for message in messages %}
           <li{% if message.tags %} class="{{ message.tags }}"{% endif %}>{{ message }}</li>
```
Comment (NutthanichN) [inline quizer_game/templates/quizer_game/edit-question-set.html]: Should it be `<li>`? I think `>` is missing.

File: quizer_game/templates/quizer_game/edit-question-set.html [hunk 2]
```diff
@@ -69,11 +70,12 @@ <h1>Edit Quiz</h1>
...
   </div>
 
   <div id="back">
-    <a href=>Back</a>
+    <script>
+      document.write('<a href="' + document.referrer + '">Back</a>');
+    </script>
   </div>
 
 </body>
```
Comment (NutthanichN) [inline quizer_game/templates/quizer_game/edit-question-set.html]: Why `<body>` and `</body>` are not in the same level?

File: quizer_game/templates/quizer_game/edit-question-set.html [hunk 3]
```diff
@@ -36,20 +37,20 @@ <h1>Edit Quiz</h1>
...
-          <label class = "question">Question {{forloop.counter}}</label>
-          <input name="question_text_{{forloop.counter}}"  value= "{{i}}" class="question_input">
-        </article>
-	    {% for j in i.choice_set.all %}
-          <article class="field">
-		    <label class = "choice">Choice {{forloop.counter}}</label>
-	        <input  name="{{forloop.parentloop.counter}}_choice_text_{{forloop.counter}}"  value= "{{j}}" class = "choice_input">
-	      </article>
-	    {% endfor %}
+    {% for i in quiz.question_set.all%}
```
Comment (Wijantra) [inline quizer_game/templates/quizer_game/edit-question-set.html]: Indent of this line should not be the same as line 37

File: quizer_game/templates/quizer_game/leaderboard.html [hunk 4]
```diff
@@ -1,66 +1,57 @@
...
+  	  </thead>
+  	  <tbody>
+	      {% for player in players %}
+	        <tr>
+            <td>{{ forloop.counter }}</td>
+            <td>{{ player.name }}</td>
+            <td>{{ player.time }}</td>
+            <td>{{ player.total_answer }}</td>
+	        {% endfor %}
+        </tr>
```
Comment (NutthanichN) [inline quizer_game/templates/quizer_game/leaderboard.html]: Shouldn't `<tr>` and `</tr>` be at the same level?

File: quizer_game/templates/quizer_game/edit-question-set.html [hunk 5]
```diff
@@ -36,20 +37,20 @@ <h1>Edit Quiz</h1>
...
+    <article class="field">
+      <label class = "question">Question {{forloop.counter}}</label>
+      <input name="question_text_{{forloop.counter}}"  value= "{{i}}" class="question_input">
+    </article>
+    {% for j in i.choice_set.all %}
```
Comment (Wijantra) [inline quizer_game/templates/quizer_game/edit-question-set.html]: Indent of this line should not be the same as line 37

**Comment by theethaj** [comment]
I've already improved them all. Please review again.

File: quizer_game/templates/quizer_game/edit-question-set.html [hunk 6]
```diff
@@ -78,5 +77,5 @@ <h1>Edit Quiz</h1>
     </script>
   </div>
 
-</body>
+  </body>
```
Comment (Wijantra) [inline quizer_game/templates/quizer_game/edit-question-set.html]: everything between <body> and </body> should decrease 1 level.

File: quizer_game/templates/quizer_game/leaderboard-index.html [hunk 7]
```diff
@@ -57,9 +56,8 @@ <h2></h2>
     </table>
   </div>
 
-  <div id="quit">
+  <div id="back">
     <a href="{% url 'quizer_game:index' %}">Back</a>
-
   </div>
   </body>
```
Comment (Wijantra) [inline quizer_game/templates/quizer_game/leaderboard-index.html]: everything between <body> and </body> should decrease 1 level.

File: quizer_game/templates/quizer_game/leaderboard.html [hunk 8]
```diff
@@ -1,66 +1,57 @@
...
-  
-  <div id="home">
-    <a href="{% url 'quizer_game:index' %}">Home</a>
+
+  <div id="back">
+    <a href="{% url 'quizer_game:leaderboard-index' %}" >Back</a>
   </div>
-  
+
   </body>
```
Comment (Wijantra) [inline quizer_game/templates/quizer_game/leaderboard.html]: everything between <body> and </body> should decrease 1 level.

File: quizer_game/templates/quizer_game/result.html [hunk 9]
```diff
@@ -109,7 +104,7 @@ <h5>Like this quiz?
   <script src="{% static 'quizer_game/scripts/js/main.js' %}"></script>
 
   <script>
-			sessionStorage.clear();
+		sessionStorage.clear();
   </script>
   
   </body>
```
Comment (Wijantra) [inline quizer_game/templates/quizer_game/result.html]: everything between <body> and </body> should decrease 1 level.

File: quizer_game/templates/quizer_game/result.html [hunk 10]
```diff
@@ -13,10 +13,10 @@
 
   </head>
   <body id="page-top">
-    <!-- END nav -->
-<!--    <h6>Total visit of quiz: {{ quiz.total_player }}</h6>-->
```
Comment (Wijantra) [inline quizer_game/templates/quizer_game/result.html]: you should delete this comment

File: quizer_game/templates/quizer_game/create-question.html [hunk 11]
```diff
@@ -71,16 +71,16 @@ <h1>Create Quiz</h1>
...
-   <a href=>Back</a>
- </div>
+  <div id="back">
+    <script>
+      document.write('<a href="' + document.referrer + '">Back</a>');
+    </script>
+  </div>
 
-</body>
+  </body>
```
Comment (Wijantra) [inline quizer_game/templates/quizer_game/create-question.html]: everything between and should decrease 1 level.

**Comment by theethaj** [comment]
I'm already improved them all. Please review again.

**Pull request merged**